### PR TITLE
Fix more ci bugs and increase logging.

### PR DESCRIPTION
**Pull request by neumantm**

### CODE REVIEW

File: scriptsAndTools/ciDoVersionBumpIfRequested.sh [hunk 1]
```diff
@@ -68,12 +68,17 @@ newVersion="$versionFirstPart.$versionLastPartIncreased-Snapshot"
...
 mvn -s "$dir/m2settings.xml" deploy -Drepo.login=$REPO_LOGIN -Drepo.pwd=$REPO_PWD
 
+set -x
+
 git tag "$version"
 
 "$dir/versionBumpLocal.sh" "$newVersion"
 git commit -a -m "Prepare for develoment on $newVersion"
 
+ssh-add -l
```
Comment (mee4895) [inline scriptsAndTools/ciDoVersionBumpIfRequested.sh]: Why do you do that? Doesn't this only clutter the logs?

File: scriptsAndTools/ciDoVersionBumpIfRequested.sh [hunk 2]
```diff
@@ -68,12 +68,17 @@ newVersion="$versionFirstPart.$versionLastPartIncreased-Snapshot"
...
+ssh git@github.com
+git fetch
```
Comment (mee4895) [inline scriptsAndTools/ciDoVersionBumpIfRequested.sh]: Does this really work? What am I missing here?

File: scriptsAndTools/ciDoVersionBumpIfRequested.sh (hunk 1 above)
Comment (neumantm) [reply scriptsAndTools/ciDoVersionBumpIfRequested.sh]: This is for log verbosity, because there are some problems with github authentication.

Comment (neumantm) [reply scriptsAndTools/ciDoVersionBumpIfRequested.sh]: It can be removed in the future once it works reliably

Comment (neumantm) [reply scriptsAndTools/ciDoVersionBumpIfRequested.sh]: But logs are automatically folded anyway...

File: scriptsAndTools/ciDoVersionBumpIfRequested.sh (hunk 2 above)
Comment (neumantm) [reply scriptsAndTools/ciDoVersionBumpIfRequested.sh]: ssh-add -l
    ssh git@github.com
 is for verbose logging
` git fetch ` should work

**Pull request merged**